src/main/config/config.h

```diff
@@ -55,10 +55,11 @@ typedef struct systemConfig_s {
     uint8_t cpu_overclock;
     uint8_t powerOnArmingGraceTime; // in seconds
     char boardIdentifier[sizeof(TARGET_BOARD_IDENTIFIER) + 1];
-    uint8_t hseMhz; // Not used for non-F4 targets
-    uint8_t configurationState; // The state of the configuration (defaults / configured)
+    uint8_t hseMhz;                 // Only used for F4 and G4 targets
+    uint8_t configurationState;     // The state of the configuration (defaults / configured)
     uint8_t schedulerOptimizeRate;
     uint8_t enableStickArming; // boolean that determines whether stick arming can be used
+    uint16_t cpu_clock_mhz;         // Only used for G4 targets
```

Comment:
Should this be in here - it does not seem to be used?

Reply:
Good find. It was used in my experimental code that allowed fine grain clock settings, but not used anymore as I settled to predefined method (F4/F7 style).